multisrc/overrides/mangathemesia/mangaswat/src/MangaSwat.kt

```diff
@@ -41,6 +43,21 @@ class MangaSwat : MangaThemesia(
     override fun headersBuilder(): Headers.Builder = super.headersBuilder()
         .add("Referer", "$baseUrl/")
 
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val request = super.searchMangaRequest(page, query, filters)
+        val urlBuilder = request.url.newBuilder()
+
+        if (query != "") {
```

Comment:
A more Kotlin-ish version.

```suggestion
        if (query.isNotEmpty()) {
```

Reply:
Why not `isNotBlank`

Reply:
Depends on how the site handles whitespace, will check this evening (UTC)